src/Extensions/Script/Serialization/CustomSerializerJsonConverter.cs

```diff
@@ -0,0 +1,39 @@
+// MIT License.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace System.Web.Script.Serialization;
+
+internal sealed class CustomSerializerJsonConverter(JavaScriptSerializer serializer) : JsonConverter<object>
```

Comment:
does this need to be different from the CustomSerializerJsonConverter?

Reply:
Also, wouldn't this have a cyclic dependency if we're passing in the JavascriptSerializer to it?

Reply:
Why there would be cyclic dependency ? We are not doing anything with the JavascriptSerializer other than calling the converter (provided there is a converter for the type). LMK if I need further clarifications.

Reply:
feel free to disregard that comment if it's not applicable. I said that since we use the converter in the JavaScriptSerializer and the converter uses it - but I guess if it's working it's not going to go recursive